aten/src/ATen/native/native_functions.yaml

```diff
@@ -8807,10 +8807,10 @@
   dispatch:
     CPU, CUDA: linalg_vector_norm_out
 
-- func: linalg_svd.U(Tensor self, bool full_matrices=True, bool compute_uv=True, *, Tensor(a!) U, Tensor(b!) S, Tensor(c!) V) -> (Tensor(a!) U, Tensor(b!) S, Tensor(c!) V)
+- func: linalg_svd.U(Tensor self, bool full_matrices=True, *, Tensor(a!) U, Tensor(b!) S, Tensor(c!) V) -> (Tensor(a!) U, Tensor(b!) S, Tensor(c!) V)
```

Comment:
These changes would break any torchscripted model using torch.linalg.svd, so I'll need to validate that's not an issue internally

a solution would be to add back compute_uv without documenting it and go through a more complete deprecation process

Reply:
Is there any news on that?
`torch.svd` still has the `compute_uv` argument, if that's needed for some reason, and we have `torch.linalg.svdvals` now.

Reply:
I contacted an internal team and they'd like svdvals to be differentiable so they can switch to it, which seems like a reasonable request? From a project perspective it looks like that's happening soon, too, so we shouldn't be held up satisfying them